tomviz/OperatorPython.cxx

```diff
@@ -377,4 +426,15 @@ EditOperatorWidget* OperatorPython::getEditorContents(QWidget* p)
 {
   return new EditPythonOperatorWidget(p, this);
 }
+
+void OperatorPython::createNewChildDataSource(
+  const QString& label, vtkSmartPointer<vtkSMProxy> producerProxy)
+{
+
+  DataSource* childDS = new DataSource(
```

Comment:
Yep, I had removed 'this' as the parent as I was running into thread affinity problems, but never put it back.